docs/guide/sending-email.md

```diff
@@ -0,0 +1,226 @@
+      sendmail: true,
+      // changes the path to the `sendmail` app
+      path: 'user/sbin/sendmail'
+    }
+  }
+};
+
+```
+<template v-slot:caption>
+/modules/@apostrophecms/email
+</template>
+
+</AposCodeBlock>
+
+## Configuring using generic SMTP
+
+Using a 3rd-party email delivery provider like Gmail, or Mailgun will probably better guarantee delivery without the email being either bounced or sent to spam. Most providers use SMTP. In fact, the `sendmail` server application uses SMTP. It just performs most of the configuration for you automatically. To configure sending through other providers, you typically need to supply three different parameters to the `nodemailer` option through an object - the 'host', 'port', and 'auth' parameters. Depending on the host there are a number of other options that could be passed as outlined in the [`nodemailer` documentation](https://nodemailer.com/smtp/).
+
+### Example
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  options: {
+    nodemailer: {
+      host: "smtp.example.com",
+      port: 587,
+      auth: {
+        user: "username",
+        pass: "password"
+      }
+    }
+  }
+};
+
+```
+<template v-slot:caption>
+/modules/@apostrophecms/email
+</template>
+
+</AposCodeBlock>
+
+## Using specific service APIs
+
+The `nodemailer` app has three built-in transports - `sendmail`, `SES` - for sending by AWS SES, and `stream` - which is used for testing. You can create your own custom transport using the `nodemailer` [transport API documentation](https://nodemailer.com/plugins/create/#transports), or use one of the many transport plugins that are available. These can either be found on the providers site, or by [searching NPM](https://www.npmjs.com/search?q=nodemailer%20transport). Each transporter will have slightly different parameters configured through an object that is then passed along with the specific transport.
+
+### Example for Mailgun
+
+<AposCodeBlock>
+
+```js
+const mg = require('nodemailer-mailgun-transport');
+
+const auth = {
+  auth: {
+    // key supplied with your Mailgun account
+    api_key: 'key-1234123412341234',
+    //one of your domains listed at your https://app.mailgun.com/app/sending/domains
+    domain: 'sandbox.domain.com'
+  }
+};
+
+module.exports = {
+  options: {
+    nodemailer: mg(auth);
+  }
+};
+
+```
+<template v-slot:caption>
+/modules/@apostrophecms/email
+</template>
+
+</AposCodeBlock>
+
+## Sending email from a module
+Once the `@apostrophecms/email` module is configured, email can be sent from any module using the `self.email(req, template, data, options)` method.
+
+The first parameter passed to this method is the page `req`.
+
+The next parameter, `template`, takes the name of a Nunjucks template that will make up the body of the email. This template should be located in the `views` template of the module. The method will pass this template, as well as an automatically generated version, to the `options` parameter of the `nodemailer` app.
```

Comment:
What does this mean?

"as well as an automatically generated version"

Reply:
it's not the nodemailer app, it's the (something) method of the nodemailer transport...

Reply:
Somehow left "plain text" out. Changed. Last sentence re-written.